drf_jsonapi/mixins.py

```diff
@@ -188,7 +192,8 @@ def retrieve(self, request, *args, **kwargs):
             resource,
             only_fields=request.fields,
             include=request.include,
-            page_size=self.request.GET.get('page[size]', settings.DEFAULT_PAGE_SIZE)
+            page_size=self.request.GET.get('page[size]', settings.DEFAULT_PAGE_SIZE),
```

Comment:
Again, why are we referencing `self.request`, when we're passing `request` as an argument?

Also, do we need to pass an element of the request (`page[size]`) when we're passing the request in as `context`?

Reply:
I will remove the `self.`

`page_size` is a discreet property that we are pulling from the request but we could just as easily hard-code it or specify some other way. By only passing the request we are leaving the serializer to have to figure out how to get it from the request (not the serializer's job).